### PR TITLE
#158755792  improve image gallery

### PR DESCRIPTION
**Pull request by Annmary12**
#### What does this PR do?
Improve image gallery

#### Description of Task to be completed?
Change the background color of product detail page
Change the `Add to cart` button color
Implement image hover for a clearer view of the product image

#### How should this be manually tested?
clone the project
cd into the project
run `meteor npm install` in your terminal
run `reaction`
check  http://localhost:3000/

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
Story Id: 158755792
Story Type: Feature
Story: Improve image gallery

#### Screenshots (if appropriate)
screen shot 2018-07-18 at 12 00 09 pmscreen shot 2018-07-18 at 3 00 13 pm

### CODE REVIEW

File: client/templates/layout.html [hunk 1]
```diff
@@ -9,6 +9,6 @@
       {{> Template.dynamic template=template}}
     </div>
 
-    <footer class="reaction-navigation-footer">{{> Template.dynamic template=layoutFooter}}</footer>
+    <!-- <footer class="reaction-navigation-footer">{{> Template.dynamic template=layoutFooter}}</footer> -->
```
Comment (abdulfataiaka) [inline client/templates/layout.html]: Please remove commented codes

File: imports/plugins/core/layout/client/components/coreLayout.js [hunk 2]
```diff
@@ -6,7 +6,8 @@ import Blaze from "meteor/gadicc:blaze-react-component";
 import { Template } from "meteor/templating";
 
 const CoreLayout = ({ actionViewIsOpen, structure }) => {
-  const { layoutFooter, template } = structure || {};
+  const { template } = structure || {};
+  // const { layoutFooter, template } = structure || {};
```
Comment (abdulfataiaka) [inline imports/plugins/core/layout/client/components/coreLayout.js]: Please remove commented codes

File: imports/plugins/core/layout/client/components/coreLayout.js [hunk 3]
```diff
@@ -25,9 +26,9 @@ const CoreLayout = ({ actionViewIsOpen, structure }) => {
         </main>
       }
 
-      { Template[layoutFooter] &&
-        <Blaze template={layoutFooter} className="reaction-navigation-footer footer-default" />
-      }
+      {/* { Template[layoutFooter] &&
+        // <Blaze template={layoutFooter} className="reaction-navigation-footer footer-default" />
+      } */}
```
Comment (abdulfataiaka) [inline imports/plugins/core/layout/client/components/coreLayout.js]: Please remove commented codes

File: imports/plugins/core/layout/client/templates/layout/admin/admin.html [hunk 4]
```diff
@@ -21,11 +21,11 @@
...
       <!-- end template region -->
 
-      <footer class="reaction-navigation-footer footer-default">
+      <!-- <footer class="reaction-navigation-footer footer-default"> -->
         <!-- begin layoutFooter -->
-        {{> Template.dynamic template=layoutFooter}}
+        <!-- {{> Template.dynamic template=layoutFooter}} -->
         <!-- end layoutFooter -->
-      </footer>
+      <!-- </footer> -->
```
Comment (abdulfataiaka) [inline imports/plugins/core/layout/client/templates/layout/admin/admin.html]: Please remove commented codes

File: imports/plugins/custom/homepage/client/components/carousel.js [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+              <span className="sr-only">Next</span>
+            </a>
+          </div>
+        </div>
+      </div>
+    );
+  }
+}
+
+export default Carousel;
```
Comment (abdulfataiaka) [inline imports/plugins/custom/homepage/client/components/carousel.js:54]: Please remove all commented codes

File: imports/plugins/custom/homepage/client/templates/footer.html [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+        </div>
+      </div>
+      <div class="container">
+        <div class="copyright">
+          &copy; Copyright
+          <strong>Andela Valhalla Team</strong>. All Rights Reserved
+        </div>
+      </div>
+    </footer>
+  </template>
```
Comment (abdulfataiaka) [inline imports/plugins/custom/homepage/client/templates/footer.html]: Add a new line after line `70` here

File: imports/plugins/custom/homepage/client/templates/landing.html [hunk 7]
```diff
@@ -0,0 +1,6 @@
+<template name="Homepage">
+  <div>
+   {{> React Home}}
+  </div>
+   {{> footer }}
+</template>
```
Comment (abdulfataiaka) [inline imports/plugins/custom/homepage/client/templates/landing.html]: Add a new line after line `6` here

**Comment by abdulfataiaka** [comment]
@Annmary12, kindly go through the file changes and correct as necessary and specified by the comments

**Comment by abdulfataiaka** [comment]
@Annmary12, ensure to squash all commits into one, and let the commit message follow convection 🙇

File: client/templates/layout.html (hunk 1 above)
Comment (Annmary12) [reply client/templates/layout.html]: I will do that now. Thanks

**Pull request merged**